sdk/core/core-http/src/policies/proxyPolicy.ts

```diff
@@ -97,7 +136,7 @@ export class ProxyPolicy extends BaseRequestPolicy {
   }
 
   public sendRequest(request: WebResourceLike): Promise<HttpOperationResponse> {
-    if (!request.proxySettings) {
+    if (!isBypassed(request.url) && !request.proxySettings) {
```

Comment:
```suggestion
    if (!request.proxySettings && !isBypassed(request.url)) {
```

Might as well do the check in this order to save time since it'll be faster to check if there are already request specific proxySettings than to consult the bypass list.